platform/linux-dpdk/Makefile.am

```diff
@@ -196,6 +197,7 @@ noinst_HEADERS = \
 		  ${top_srcdir}/platform/linux-generic/include/odp_packet_socket.h \
 		  ${top_srcdir}/platform/linux-generic/include/odp_pkt_queue_internal.h \
 		  ${srcdir}/include/odp_pool_internal.h \
+		  ${top_srcdir}/platform/linux-generic/include/odp_pool_subsystem.h \
```

Comment:
Yes, it should be moved out. The current decision is that we will make the directory changes after SFO17. But, the files should be split correctly already. If keeping them where they are currently is resulting in problems, we can move them now.